docs/modules/ROOT/examples/advanced-networking-interfaces.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast-1
```

Comment:
It's not a big deal but generally we named our CR's as the CR kind name (**hazelcast**) or CR kind name - sample **(hazelcast-sample)**